.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -33,19 +33,6 @@ body:
       description: Copy and paste the version from the about page of FluentHub's settings. If you're building from GitHub, simply provide the commit hash you built on.
     validations:
       required: true
-  - type: textarea
-    attributes:
-      label: 📟 Stack Trace
-      description: Copy and paste the stack trace from FluentHub's error page.
-      value: |
-        <details><summary>Stack trace details</summary><div>
-
-        \```
-        paste here
-        \```
-        </div></details>
-    validations:
-      required: true
```

Comment:
Bad idea. I placed it right there because it was directly *related* to Version Number section.

Reply:
What do you mean 'related'. Should be related assets section more than it.

Reply:
> What do you mean 'related'. Should be related assets section more than it.

True. ok sure then.